--- mir-1.1.0/src/CMakeLists.txt.orig	2023-03-01 21:03:37.167115550 +0100
+++ mir-1.1.0/src/CMakeLists.txt	2023-03-01 21:07:55.529049216 +0100
@@ -12,7 +12,7 @@ add_subdirectory(cookie/)
 # We need MIR_CLIENT_PLATFORM_PATH in both libmirclient and the platform
 # implementations
 set(MIR_CLIENT_PLATFORM_PATH
-  ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/mir/client-platform
+  ${CMAKE_INSTALL_LIBDIR}/mir/client-platform
 )
 
 # Add the core and platform implementations before exposing any private APIs
--- mir-1.1.0/src/platforms/CMakeLists.txt.orig	2023-03-01 21:03:37.167115550 +0100
+++ mir-1.1.0/src/platforms/CMakeLists.txt	2023-03-01 21:08:34.138840049 +0100
@@ -21,7 +21,7 @@ set(MIR_INPUT_PLATFORM_VERSION_SCRIPT
 set(MIR_INPUT_PLATFORM_VERSION_SCRIPT ${MIR_INPUT_PLATFORM_VERSION_SCRIPT} PARENT_SCOPE)
 
 set(MIR_SERVER_PLATFORM_PATH
-  ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/mir/server-platform
+  ${CMAKE_INSTALL_LIBDIR}/mir/server-platform
 )
 set(MIR_SERVER_PLATFORM_PATH
   ${MIR_SERVER_PLATFORM_PATH}
