Disable functionality which depends on Ubuntu changes in libinput.
--- mir-0.21.0/src/platforms/evdev/libinput_device.cpp.orig	2016-04-08 16:03:06.000000000 +0200
+++ mir-0.21.0/src/platforms/evdev/libinput_device.cpp	2016-04-13 21:02:02.006277136 +0200
@@ -290,11 +290,11 @@
     uint32_t const height = screen.size.height.as_int();
 
     data.action = action;
-    data.pressure = libinput_event_touch_get_pressure(touch);
+/*  data.pressure = libinput_event_touch_get_pressure(touch); */
     data.x = libinput_event_touch_get_x_transformed(touch, width);
     data.y = libinput_event_touch_get_y_transformed(touch, height);
-    data.major = libinput_event_touch_get_major_transformed(touch, width, height);
-    data.minor = libinput_event_touch_get_minor_transformed(touch, width, height);
+/*  data.major = libinput_event_touch_get_major_transformed(touch, width, height);
+    data.minor = libinput_event_touch_get_minor_transformed(touch, width, height); */
 }
 
 void mie::LibInputDevice::handle_touch_motion(libinput_event_touch* touch)
