--- mir-0.16.0/src/platforms/mesa/server/kms/cursor.cpp.orig	2015-03-27 03:02:23.000000000 +0100
+++ mir-0.16.0/src/platforms/mesa/server/kms/cursor.cpp	2015-04-22 21:44:29.876555247 +0200
@@ -25,7 +25,7 @@
 #include "mir/graphics/cursor_image.h"
 
 #include <xf86drm.h>
-#include <drm/drm.h>
+#include <libdrm/drm.h>
 
 #include <boost/exception/errinfo_errno.hpp>
 
--- mir-0.26.2/tests/unit-tests/platforms/mesa/kms/test_cursor.cpp.orig	2017-03-27 04:18:32.000000000 +0200
+++ mir-0.26.2/tests/unit-tests/platforms/mesa/kms/test_cursor.cpp	2017-05-28 19:15:40.087281140 +0200
@@ -24,7 +24,7 @@
 #include "mir/graphics/cursor_image.h"
 
 #include <xf86drm.h>
-#include <drm/drm.h>
+#include <libdrm/drm.h>
 
 #include "mir/test/doubles/mock_gbm.h"
 #include "mir/test/doubles/mock_drm.h"
--- mir-0.26.2/src/platforms/eglstream-kms/server/display.cpp.orig	2017-03-27 04:18:32.000000000 +0200
+++ mir-0.26.2/src/platforms/eglstream-kms/server/display.cpp	2017-05-28 21:15:48.043865508 +0200
@@ -33,7 +33,7 @@
 #include "mir/renderer/gl/render_target.h"
 #include "mir/renderer/gl/context.h"
 
-#include <drm/drm.h>
+#include <libdrm/drm.h>
 #include <xf86drmMode.h>
 #include <sys/ioctl.h>
 #include <system_error>
--- mir-0.26.2/src/platforms/eglstream-kms/server/egl_output.cpp.orig	2017-03-27 04:18:32.000000000 +0200
+++ mir-0.26.2/src/platforms/eglstream-kms/server/egl_output.cpp	2017-05-28 21:26:01.860524941 +0200
@@ -23,7 +23,7 @@
 #include "kms-utils/kms_connector.h"
 
 #include <cstring>
-#include <drm/drm.h>
+#include <libdrm/drm.h>
 #include <sys/ioctl.h>
 #include <vector>
 #include <boost/throw_exception.hpp>
