--- mir-1.8.2/src/server/CMakeLists.txt.orig	2023-03-03 16:04:54.548072759 +0100
+++ mir-1.8.2/src/server/CMakeLists.txt	2023-03-03 16:06:12.297651553 +0100
@@ -146,7 +146,6 @@ target_link_libraries(mirserver LINK_PUB
   ${X11_XCURSOR_LDFLAGS} ${X11_XCURSOR_LIBRARIES}
   ${LTTNG_UST_LDFLAGS} ${LTTNG_UST_LIBRARIES}
   ${FREETYPE_LDFLAGS} ${FREETYPE_LIBRARIES}
-  atomic
 )
 
 target_include_directories(mirshelldecoration
