--- mir-0.16.0/tests/acceptance-tests/test_surface_morphing.cpp.orig	2015-09-29 07:33:42.000000000 +0200
+++ mir-0.16.0/tests/acceptance-tests/test_surface_morphing.cpp	2015-09-29 18:49:35.461087424 +0200
@@ -16,6 +16,9 @@
  * Authored By: Alan Griffiths <alan@octopull.co.uk>
  */
 
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+
 #include "mir/scene/null_surface_observer.h"
 #include "mir/scene/surface.h"
 
--- mir-0.16.0/tests/acceptance-tests/test_surface_specification.cpp.orig	2015-09-29 07:33:42.000000000 +0200
+++ mir-0.16.0/tests/acceptance-tests/test_surface_specification.cpp	2015-09-29 18:59:49.114395005 +0200
@@ -16,6 +16,9 @@
  * Authored By: Alan Griffiths <alan@octopull.co.uk>
  */
 
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+
 #include "mir/events/event_builders.h"
 #include "mir/scene/null_surface_observer.h"
 #include "mir/scene/surface.h"
--- mir-0.16.0/tests/unit-tests/client/test_connection_resource_map.cpp.orig	2015-09-29 07:33:42.000000000 +0200
+++ mir-0.16.0/tests/unit-tests/client/test_connection_resource_map.cpp	2015-09-29 20:34:08.974157185 +0200
@@ -16,10 +16,11 @@
  * Authored by: Kevin DuBois <kevin.dubois@canonical.com>
  */
 
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
 #include "src/client/connection_surface_map.h"
 #include "src/client/mir_surface.h"
 #include "mir/test/doubles/mock_client_buffer_stream.h"
-#include <gtest/gtest.h>
 
 namespace mf = mir::frontend;
 namespace mcl = mir::client;
--- mir-0.16.0/tests/unit-tests/compositor/test_dropping_schedule.cpp.orig	2015-09-29 07:33:42.000000000 +0200
+++ mir-0.16.0/tests/unit-tests/compositor/test_dropping_schedule.cpp	2015-09-29 20:59:31.804093576 +0200
@@ -16,11 +16,12 @@
  * Authored by: Kevin DuBois <kevin.dubois@canonical.com>
  */
 
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
 #include "mir/frontend/client_buffers.h"
 #include "src/server/compositor/dropping_schedule.h"
 #include "mir/test/doubles/stub_buffer.h"
 #include "mir/test/fake_shared.h"
-#include <gtest/gtest.h>
 
 using namespace testing;
 namespace mtd = mir::test::doubles;
--- mir-0.16.0/tests/unit-tests/compositor/test_queueing_schedule.cpp.orig	2015-09-29 07:33:42.000000000 +0200
+++ mir-0.16.0/tests/unit-tests/compositor/test_queueing_schedule.cpp	2015-09-29 21:21:12.014039012 +0200
@@ -16,9 +16,10 @@
  * Authored by: Kevin DuBois <kevin.dubois@canonical.com>
  */
 
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
 #include "src/server/compositor/queueing_schedule.h"
 #include "mir/test/doubles/stub_buffer.h"
-#include <gtest/gtest.h>
 
 using namespace testing;
 namespace mtd = mir::test::doubles;
--- mir-0.16.0/tests/unit-tests/graphics/test_platform_prober.cpp.orig	2015-09-29 07:33:42.000000000 +0200
+++ mir-0.16.0/tests/unit-tests/graphics/test_platform_prober.cpp	2015-09-29 21:52:25.397293726 +0200
@@ -17,6 +17,7 @@
  */
 
 #include <gtest/gtest.h>
+#include <gmock/gmock.h>
 
 #include "mir/graphics/platform.h"
 #include "mir/graphics/platform_probe.h"
--- mir-0.16.0/tests/unit-tests/input/evdev/test_evdev_device_detection.cpp.orig	2015-09-29 07:33:42.000000000 +0200
+++ mir-0.16.0/tests/unit-tests/input/evdev/test_evdev_device_detection.cpp	2015-09-30 17:44:16.817626009 +0200
@@ -22,6 +22,7 @@
 #include "mir_test_framework/udev_environment.h"
 
 #include <gtest/gtest.h>
+#include <gmock/gmock.h>
 #include <tuple>
 
 namespace mtf = mir_test_framework;
--- mir-0.16.0/tests/unit-tests/input/test_input_event.cpp.orig	2015-09-29 07:33:42.000000000 +0200
+++ mir-0.16.0/tests/unit-tests/input/test_input_event.cpp	2015-09-30 18:04:45.110907797 +0200
@@ -17,6 +17,7 @@
  */
 
 #include <gtest/gtest.h>
+#include <gmock/gmock.h>
 
 #include "mir/events/event_private.h"
 #include "mir_toolkit/events/input/input_event.h"
--- mir-0.16.0/tests/unit-tests/dispatch/test_readable_fd.cpp.orig	2015-09-29 07:33:42.000000000 +0200
+++ mir-0.16.0/tests/unit-tests/dispatch/test_readable_fd.cpp	2015-09-30 19:12:50.107403032 +0200
@@ -19,6 +19,7 @@
 #include "mir/dispatch/readable_fd.h"
 
 #include <gtest/gtest.h>
+#include <gmock/gmock.h>
 
 namespace md = mir::dispatch;
 
