--- mir-0.13.3/tests/acceptance-tests/test_surface_placement.cpp.orig	2015-06-22 07:30:08.000000000 +0200
+++ mir-0.13.3/tests/acceptance-tests/test_surface_placement.cpp	2015-06-27 16:22:34.473352934 +0200
@@ -16,6 +16,9 @@
  * Authored By: Alan Griffiths <alan@octopull.co.uk>
  */
 
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
 #include "mir_test_framework/connected_client_headless_server.h"
 
 #include "mir/events/event_builders.h"
